packages/edit-site/src/components/global-styles/screen-revisions/get-revision-changes.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const translationMap = {
+	blocks: __( 'Block styles' ),
```

Comment:
Can we break out block names in a follow up? For example show "Paragraph block typography" etc instead of a generic "Block styles". I think block styles are going to change a bunch and a lot of "Block styles" being listed is not super useful.

Reply:
For blocks it makes sense. I think we have to put a limit on it at some point as it needs to be calculated for every revision item on render.

I'll see how memoizing/caching each one might work too. 

There was an exploration into comparing the style/settings tree deeply: https://github.com/WordPress/gutenberg/pull/52829#issuecomment-1659027089